lib/cc/analyzer/engine.rb

```diff
@@ -34,8 +34,12 @@ def run(stdout_io, container_listener)
         )
 
         container.on_output("\0") do |output|
-          unless output_filter.filter?(output)
-            stdout_io.write(output) || container.stop
+          next if output.blank?
+
+          issue_hash = IssueHash.new(output)
```

Comment:
Mentioned IRL, but for visibility:

I think we should go with an actual `Issue` class we can pass around and filter on that responds to methods. Then we `#to_json` that for the `stdout_io.write` call.